apps/web/app/(app)/environments/[environmentId]/settings/(organization)/members/actions.ts

```diff
@@ -167,6 +168,11 @@ export const inviteUserAction = async (
     throw new AuthenticationError("Not authenticated");
   }
 
+  const user = await getUser(session.user.id);
+  if (!user) {
+    throw new AuthenticationError("User not found");
```

Comment:
sometimes you throw `AuthenticationError`, sometimes a simple `Error`. We should be consistent.